frontend/javascripts/oxalis/model/reducers/volumetracing_reducer_helpers.ts

```diff
@@ -179,6 +182,23 @@ export function setMappingNameReducer(
   if (mappingType !== "HDF5" || !isMappingEnabled) {
     mappingName = null;
   }
+  if (mappingName) {
+    const { agglomerates, mappings } = getSegmentationLayerByName(
+      state.dataset,
+      volumeTracing.tracingId,
+    );
+    const listToSearchIn = mappingType === "HDF5" ? agglomerates : mappings;
+    if (
+      mappingType === "HDF5" &&
+      (listToSearchIn == null || listToSearchIn.indexOf(mappingName) === -1)
+    ) {
+      console.error(
+        `Cannot set mapping ${mappingName} of type ${mappingType} for volume tracing ${volumeTracing.tracingId} because it is not included in the agglomerate list of the data layer: ${agglomerates}.`,
```

Comment:
```suggestion
        `Cannot set mapping ${mappingName} of type ${mappingType} for volume tracing ${volumeTracing.tracingId} because it is not included in the mapping list of the data layer: ${listToSearchIn}.`,
```